test/ApplicationInsightsJavaScriptSnippetTest/ApplicationInsightsJavaScriptSnippetTest.cs

```diff
@@ -78,8 +78,14 @@ public ApplicationInsightsJavaScriptSnippetTest(ITestOutputHelper output) : base
 
                     var validator = new Validator(httpClient, httpClientHandler, logger, deploymentResult);
 
-                    logger.LogInformation("Verifying home page");
-                    await validator.VerifyScriptCheckPage(response);
+                    logger.LogInformation("Verifying layout page");
+                    await validator.VerifyLayoutPage(response);
+
+                    logger.LogInformation("Verifying layout page before script");
+                    await validator.VerifyLayoutPageBeforeScript(response);
+
```

Comment:
I didn't remove the test that verified the entire layout page (including the script with the instrumentation key). I just added more to also test the relative position. I can delete all the extra stuff from the script and just check for the instrumentation key in that test instead.